Packs/Base/Scripts/GetErrors/GetErrors.py

```diff
@@ -0,0 +1,44 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_errors(entries):
+    """
+        entries should be a list of demisto entries
+
+        :type entries: ``list``
+        :param entries: multiples entries of results of demisto.executeCommand()
+
+        :return: Error message extracted from the demisto.executeCommand() result
+        :rtype: ``string``
+    """
+    error_messages = []
+    for entry in entries:
+        assert len(entry) == 1
+        entry_details = entry[0]
+        is_error_entry = isinstance(entry_details, dict) and entry_details['Type'] == entryTypes['error']
+        if is_error_entry:
+            demisto.debug(f'error entry contents: "{entry_details["Contents"]}"')
+            error_messages.append(entry_details['Contents'])
+
+    return error_messages
+
+
+def main():
+    try:
+        args = demisto.args()
+        entry_ids = args.get('entry_id')
```

Comment:
please use `argToList(args.get('entry_id'))` in case someone will pass a string .
Can you load the `lastCompletedTaskEntries` from `demisto.context()` if entry_id was not provided?

Reply:
I was using `argToList` initially and it was causing errors for me. I'll try again.